### ISSUE TITLE
(3) [ EPIC ] Figma file updates: Update get.gov Base Pages

### ISSUE THREAD

**Issue by rileyorr** [cisagov/manage.get.gov#1973]
**Update from Aug 2024 Audit** 

### Issue description

The audit form Aug 2024 found that the Get.gov base pages do not match production or are missing.  Base pages are mockups of production pages so that designers can use for new feature updates without having to recreate the "base" for every new update. 

Now that the design system component updates have been complete, these pages can be updates.  
Notes on what needs to be updates are made directly in the Figma file. 

The work will be broken up into multiple issues one for each page for easier Figma implementation. The pages with their story links are listed below:

- [ ] get.gov/Home  [Figma page](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=2349-8658)  
- [ ] get.gov/Domains - F[igma page](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=2196-35124) 
- [ ] get.gov/Blog  - [Figma page ](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=2191-34071)
- [ ] get.gov/about  - [Figma page ](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=9045-131095)
- [ ] get.gov/help  - [Figma page  ](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=2202-24244)
- [ ] get.gov/SearchResults  - [Figma page](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=2351-45530) 
- [ ] get.gov/HTML-Errors  - [Figma page](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=3972-37221)

### Acceptance criteria
- [ ] Figma get.gov base page matches production
- [ ] The [Get.gov cover page](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=2211-25393) is updated
- [ ] Local get.gov components are centralized on the[ get.gov component page](https://www.figma.com/design/qeWM03sfjXgHBHB23rsD6X/get.gov?node-id=11270-4799)
- [ ] Auto-layout is used when possible.
- [ ] Layers are organized and named
- [ ] Design system components and styles are used when possible
- [ ] Changes are made in a Figma branch and are reviewed by another designer
- [ ] Changes are merged

### Additional context
This audit was done in Aug 2024 and should hopefully be completed this year before updates occur and the audit becomes outdated. If this work trickles into 2025 make sure to confirm comments against production again. 

### Issue description

The audit form Aug 2024 found that the Registar - get.gov/manage/forms base pages do not match production or are missing.  Base pages are mockups of production pages so that designers can use for new feature updates without having to recreate the "base" for every new update. 

Now that the design system component updates have been complete, these pages can be updates.  
Notes on what needs to be updates are made directly in the Figma file. 

The work will be broken up into multiple issues one for each page for easier Figma implementation. The pages with their story links are listed below:

### Links to other issues
This work is blocked by all Figma Design System stories. 

Issues relating to this epic are:
- #2781 
- #2782 
- #2783 
- #2784 
- #2785 
- #2786 
- #2787

**Comment by rileyorr** [comment]
@gabydisarli and I met and decided that not all the missing pages should be mocked up. This issue's description was updated on 4/30 to reflect the work to be done.

**Comment by joet333** [comment]
@Katherine-Osos when I did the audit, the pages for all of production in registrar and info site were noted and added.. Those pages have not been merged or updated, but have been noted with what updates were needed.  The new files also all have new cover pages. 

Did we want to update this issue?

**Comment by joet333** [comment]
@Katherine-Osos 
TLDR: Links go to the base pages (mockups of production for reuse on new feature/updates/prototypes) and on the base page are updates that will match production.

----------------------
I would suggest this would be an epic and then I would have an issue for each page (see links) 
The updates for each page from the audit are already on the page. 

The other two epics I would create besides this one would be:
1) Registar Forms Base  
2) Registar Manage Base
Note that org model should already match production since work has no yet been completed. 

I would recommend this work be complete only after the component updates that we are working on that way we can update the components to relink to the correct library as we go. I do think some of these issues can be completed in 30min where some might take a day or so. I would also recommend that this work get completed before at least the end of the year to make sure audit does not get outdated.


Let me know if this format makes sense for the updates that are required. If so I can make sure to create all the issue that we need for future.

**Comment by joet333** [comment]
Noting that we are moving forward with newly written work.

**Comment by Katherine-Osos** [comment]
Closing ticket - no longer how we're tracking the Figma work.
